src/config.rs

```diff
@@ -468,6 +468,11 @@ pub struct DbOptions {
 }
 
 impl DbOptions {
+    /// Converts the DbOptions to a JSON string representation
+    pub fn to_json_string(&self) -> Result<String, serde_json::Error> {
+        serde_json::to_string_pretty(self)
```

Comment:
I prefer non-pretty. I would like the JSON log output to go on a single line. This will make it easier to keep it atomic in some logging frameworks.